packages/dso-toolkit/src/components/modal/modal.mixins.scss

```diff
@@ -19,46 +19,65 @@
 }
 
 @mixin root() {
-  bottom: 0;
-  height: 100%;
-  left: 0;
-  position: fixed;
-  right: 0;
-  top: 0;
-  z-index: zindex.$modal;
+  // dialog
+  // ------
+  // overwrite UA styles:
+  overflow: visible;
+
+  padding: 0;
+
+  border: 0;
+
+  &::backdrop {
+    background-color: fade-out(modal-variables.$backdrop-bg, modal-variables.$backdrop-opacity);
+  }
+
+  // fullscreen optie
+  &[fullscreen] {
```

Comment:
Deze kan hier eigenlijk niet staan: De `root()` mixin wordt ook voor het Web Component gemaakt. Daar wordt ie ook aangeroepen. Samengevat wordt er deze CSS gegenereerd:

```css
:host([fullscreen]) {
  /* fullscreen CSS */
}

:host()[fullscreen] {
  /* full screen CSS -- maar deze werkt natuurlijk niet */
}
```

Los van dat het niet werkt is het ook nog 'n beetje dubbelop. Beter kunnen we de mixin alleen in de implementaties aanroepen, en niet in de root mixin.

Reply:
Check. Ik twijfelde zelf al een beetje aan deze oplossing, maar had even niks beters. Morgen in rotjeknor een leuke klus om dat om te bakken :)

Reply:
Ziet er goed uit!